### DISCUSSION TITLE
fatal: bad revision 'HEAD'

### DISCUSSION THREAD

**Discussion by hurska** [Project-MONAI/MONAI#4306] [Q&A]
Hello everyone. I had install Monai a few days ago and it was working. But I had to uninstall all my packages yesterday. I installed monai today and I cannot import it. I am recieving the following message **fatal: bad revision 'HEAD'** .  Can you please advise me how to fix it?

**Comment by wyli** [comment]
might be related to the recent tutorial branch renaming, in case it's useful here are the commands to rename https://github.com/Project-MONAI/tutorials/issues/717#issuecomment-1131490247
